assets/css/_mixins.scss

```diff
@@ -1,4 +1,6 @@
 $orange: #f5914f;
+$grey: rgba(122, 120, 120, 1);
```

Comment:
Why do you use `rgba` notation, if alpha value equals 1? You can skip it then and have usual #rrggbb hex value